R/ExtendingFibonacci.R

```diff
@@ -7,6 +7,9 @@
 #'
 #' @examples fibonacci_Extend(10)
 fibonacci_Extend <- function(n) {
```

Comment:
Minor thing, but for the `@param n` line, in general it's good to specify the input type e.g. "numeric" (I know it's obvious here :) )